app/views/admin/statistics_announcements/_filter_options.html.erb

```diff
@@ -0,0 +1,80 @@
+<%
+  anchor ||= anchor || ""
+  raise "filter action required" unless defined?(filter_action)
+%>
+
+<div class="app-view-filter govuk-!-margin-right-5 govuk-!-padding-5">
+  <%= form_with url: filter_action + anchor, method: :get do |_form| %>
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Filter by",
+      margin_bottom: 4,
+    } %>
+
+    <%= render "govuk_publishing_components/components/input", {
+      label: {
+        text: "Title or slug",
+        bold: true,
+      },
+      name: "title",
+      id: "title_filter",
+      value: @filter.options[:title],
+    } %>
+
+    <%= render "components/select-with-search", {
+      label: "Organisation",
+      id: "organisation_filter",
+      name: "organisation_id",
+      heading_size: "s",
+      grouped_options: admin_organisation_filter_options(@filter.options[:organisation_id]),
+    } %>
+
+    <%= render "govuk_publishing_components/components/select", {
+      id: "release_date_filter",
+      label: "Release date",
+      name: "dates",
+      heading_size: "s",
+      options: [
+        {
+          text: "All announcements",
+          value: nil,
+          selected: @filter.options[:dates] == nil
+        },
+        {
+          text: "Future releases",
+          value: "future",
+          selected: @filter.options[:dates] == "future"
+        },
+        {
+          text: "Next 2 weeks",
+          value: "imminent",
+          selected: @filter.options[:dates] == "imminent"
+        },
+        {
+          text: "Past announcements",
+          value: "past",
+          selected: @filter.options[:dates] == "past"
+        }
+      ]
+    } %>
+
+    <%= render "govuk_publishing_components/components/checkboxes", {
+      name: "unlinked_only",
+      heading: "Announcements",
+      no_hint_text: true,
```

Comment:
Looks like this should have `heading_size: "s"` to keep the filter headings consistent